src/views/Profile.vue

```diff
@@ -1,20 +1,21 @@
 <template>
   <div class="profile-page">
-
     <div class="user-info">
       <div class="container">
         <div class="row">
-
           <div class="col-xs-12 col-md-10 offset-md-1">
-            <img src="http://i.imgur.com/Qr71crq.jpg" class="user-img" />
-            <h4>Eric Simons</h4>
+            <img :src="profile.image" class="user-img"/>
+            <h4>{{ profile.username }}</h4>
             <p>
-              Cofounder @GoThinkster, lived in Aol's HQ for a few months, kinda looks like Peeta from the Hunger Games
+              {{ profile.bio }}
             </p>
-            <button class="btn btn-sm btn-outline-secondary action-btn">
+            <button class="btn btn-sm btn-secondary action-btn" v-if="profile.following" @click.prevent="unfollow()">
+              <i class="ion-checkmark-round"></i>
+              Following {{ profile.username }}
+            </button>
+            <button class="btn btn-sm btn-outline-secondary action-btn" v-if="!profile.following" @click.prevent="follow()">
               <i class="ion-plus-round"></i>
-              &nbsp;
-              Follow Eric Simons
+              Follow {{ profile.username }}
```

Comment:
There is just missing the third option:
When the connected user is looking at his own page, the follow button becomes a link to his setting profile

<img width="817" alt="screen shot 2017-11-10 at 21 53 54" src="https://user-images.githubusercontent.com/2429708/32678276-053aea3c-c662-11e7-8182-7522b921e78d.png">

Reply:
Yes, did not want to change the auth store to help with that but it has to be done.

Reply:
Ok, I'll merge it now and wait for your next PR